src/index.html

```diff
@@ -6,7 +6,53 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="names">
+      <div class="names__day">Mon</div>
+      <div class="names__day">Tue</div>
+      <div class="names__day">Wed</div>
+      <div class="names__day">Thu</div>
+      <div class="names__day">Fri</div>
+      <div class="names__day">Sat</div>
+      <div class="names__day">Sun</div>
+    </div>
+
+    <div class="
+      calendar
+      calendar--start-day_sat
+      calendar--month-length_31
```

Comment:
```suggestion
      calendar--start-day--sat
      calendar--month-length--31
```
Значення модифікатора принято записувати подвійним тире 
https://ru.bem.info/methodology/naming-convention/#%D1%81%D1%82%D0%B8%D0%BB%D1%8C-two-dashes

Reply:
Да, но это и так часть модификатора "start-day". Однако тут есть ошибка, так как слова принято разделять с помощью "-"🙂